src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/TournamentsController.cs

```diff
@@ -280,6 +281,25 @@ public ActionResult DeleteConfirmed(int id)
             _tournamentService.Delete(id);
             return RedirectToAction("Archived");
         }
+        
+        /// <summary>
+        /// Activate tournament
+        /// </summary>
+        /// <param name="id">Identifier of tournament to activate</param>
+        /// <returns></returns>
+        public ActionResult Activate(int id)
+        {
+            try
+            {
+                _tournamentService.Activate(id);
+            }
+            catch(Exception)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtype or use exception filters instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS2221)